Lib/json/tool.py

```diff
@@ -25,24 +25,35 @@ def main():
                         help='a JSON file to be validated or pretty-printed')
     parser.add_argument('outfile', nargs='?', type=argparse.FileType('w'),
                         help='write the output of infile to outfile')
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument('--indent', default=4, type=int,
+                       help='Indent level (number of spaces) for '
+                       'pretty-printing. Defaults to 4.')
+    group.add_argument('--no-indent', action='store_const', dest='indent',
```

Comment:
Is --no-indent differen than --indent=0?

Would it be possible to use a default of 4 spaces when using --indent with no argument?